tests/regression/tests/REQUEST-932-APPLICATION-ATTACK-RCE/932236.yaml

```
@@ -838,3 +838,67 @@ tests:
             version: HTTP/1.0
           output:
             no_log_contains: id "932236"
+  - test_title: 932236-46
+    desc: "FP with word environment"
+    stages:
+      - stage:
+          input:
+            dest_addr: 127.0.0.1
+            headers:
+              User-Agent: "OWASP CRS test agent"
+              Host: localhost
+              Accept: "text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5"
+            method: GET
+            port: 80
+            uri: /get?args=environment
+            version: HTTP/1.0
+          output:
+            no_log_contains: id "932236"
+  - test_title: 932236-47
+    desc: "FP with word performance"
+    stages:
+      - stage:
+          input:
+            dest_addr: 127.0.0.1
+            headers:
+              User-Agent: "OWASP CRS test agent"
+              Host: localhost
+              Accept: "text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5"
+            method: GET
+            port: 80
+            uri: /get?args=performance
+            version: HTTP/1.0
+          output:
+            no_log_contains: id "932236"
+  - test_title: 932236-48
+    desc: "Block execution of env command with arguments"
+    stages:
+      - stage:
+          input:
+            dest_addr: 127.0.0.1
+            headers:
+              User-Agent: "OWASP CRS test agent"
+              Host: localhost
+              Accept: "text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5"
+            method: GET
+            port: 80
+            uri: /get?args=env --version
+            version: HTTP/1.0
+          output:
+            no_log_contains: id "932236"
```

From what I understand, this test should expect a match. Am I wrong?
```suggestion
            log_contains: id "932236"
```

---

No, your right that's meant to be ``log_contains:``, thanks for catching that